WooCommerce/Classes/ViewRelated/Themes/ThemesCarouselView.swift

```diff
@@ -0,0 +1,130 @@
+import SwiftUI
+import Kingfisher
+
+
+struct ThemesCarouselView: View {
+    /// Tuple array containing theme name and theme Url string pair.
+    let themesInfo: [(name: String, url: String)] = [
+        (name: "Google", url: "google.com"),
+        (name: "Facebook", url: "facebook.com"),
+        (name: "WordPress", url: "wordpress.com"),
+        (name: "Microsoft", url: "microsoft.com"),
+        (name: "Amazon", url: "amazon.com"),
+        (name: "Apple", url: "apple.com")
+    ]
+
+    /// The title of the message at the end of the carousel.
+    let lastMessageHeading: String
+
+    /// The content of the message at the end of the carousel.
+    let lastMessageContent: String
+
+
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            HStack() {
+
+                // Demo for correct state
+                ForEach(themesInfo, id: \.name) { theme in
+                    if let themeUrl = getThemeUrl(themeUrl: theme.url) {
+                        ThemeImage(url: themeUrl)
+                    }
```

Comment:
Should we also fall back to `ThemeName` if the url cannot be found?

```suggestion
                    if let themeUrl = getThemeUrl(themeUrl: theme.url) {
                        ThemeImage(url: themeUrl)
                    } else {
                        ThemeName(name: theme.name)
                    }
```

Reply:
Ah yes, I intentionally didn't add error handling here (same as there's no error handling in `ThemeImage` yet if the image can't be loaded) as this is just UI and I'm thinking to add them a separate PR. But now that I think of it, this is straightforward enough to add now.